cucumber-expressions/java/src/main/java/io/cucumber/cucumberexpressions/CucumberExpression.java

```diff
@@ -68,21 +68,16 @@ private String buildCaptureRegexp(List<String> regexps) {
     }
 
     @Override
-    public List<Argument<?>> match(String text, final DataTable argument) {
+    public List<Argument<?>> match(String text, List<List<String>> tableArgument) {
```

Comment:
I have changed the to match(String text, List<List<String>> tableArgument) so callers don't have to provide their own data table with converter which needs a parameter type registry instance. This simplifies the logistics of using Expressions.